test/E2ETests/Automation.CLI/CLITestBase.cs

```diff
@@ -172,10 +176,13 @@ public void ValidatePassedTestsContain(params string[] passedTests)
         {
             foreach (var test in passedTests)
             {
-                var testFound = this.runEventsHandler.PassedTests.Any(
+                var tests = this.runEventsHandler.PassedTests.ToList();
```

Comment:
For efficiency reasons, maybe we can move the line below outside the foreach loop to avoid calling ToList every time we iterate over a new test ?
`var tests = this.runEventsHandler.PassedTests.ToList();`

Reply:
This probably runs on max of 4 tests with one or 2 names to match. But yeah I will move it up next time I touch this code, to enumerate before we search the tests. I am calling to list just to make it easier to debug. 👍

Reply:
Or tomorrow. :)

Reply:
Done